src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,28 @@
+/* Experimental text primitive.
+ */
+var getMeshMixin = require('../getMeshMixin');
+var registerPrimitive = require('../primitives').registerPrimitive;
+var utils = require('../../../utils/');
+
+registerPrimitive('a-text', utils.extendDeep({}, getMeshMixin(), {
+  defaultComponents: {
+    'text': {anchor: 'align', width: 5}
+  },
+  mappings: {
+    align: 'text.align',
```

Comment:
Sorry. I meant two things:

1. Populate the mappings given the text component schema. Check the mesh mixin how we do it for the material properties. This allows us to not have to update the primitive every time we update a property name.

2. Use hyphens instead of all lowercase. `font-image`  vs `fontimage`